src/ExtensionsSample/Samples/EasyTableSamples.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+using Microsoft.WindowsAzure.MobileServices;
+using WebJobs.Extensions.EasyTables;
+
+namespace ExtensionsSample
+{
+    public static class EasyTableSamples
```

Comment:
Done.

---
In reply to: [54779562](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/45#discussion_r54779562) [](ancestors = 54779562)